cmd/bosun/database/silence_data.go

```diff
@@ -2,9 +2,9 @@ package database
 
 import (
 	"encoding/json"
-	"log"
 	"time"
 
+	"bosun.org/slog"
```

Comment:
go format wants to reorder these imports. Run gofmt on this file and this should be good to merge.

Reply:
Done, thanks!